examples/advanced/finance/requirements.txt

```diff
@@ -0,0 +1,7 @@
+nvflare>=2.3.0
+openmined.psi==1.1.1
+pandas
+xgboost>=1.7.0
```

Comment:
let's update this version later with xgboost example.

Reply:
sure